src/studio/src/designer/frontend/shared/features/dataModelling/DataModelling.tsx

```diff
@@ -65,7 +66,8 @@ function DataModelling(props: IDataModellingContainerProps): JSX.Element {
   const onSchemaSelected = setSelectedModelMetadata;
 
   const onSaveSchema = (schema: any) => {
-    dispatch(saveDataModel({ schema, metadata: selectedModelMetadata }));
+    const $id = sharedUrls().dataModelsApi + selectedModelMetadata?.value?.repositoryRelativeUrl;
```

Comment:
Maybe check if there is a reposityRelativeUrl before dispatching save here?
We do not want to set the id without it, and i don't think we have a repositoryRelativeUrl, before the first time we have putted a schema.

Reply:
We are actually missing a POST call on create to get the metadata of a new schema.
The API for POST hasn't been created yet.

Reply:
Until we have a repositoryRelativeUrl here, we can fallback to `'App/models'` maybe.

Reply:
```js
const $id = sharedUrls().dataModelsApi + (selectedModelMetadata?.value?.repositoryRelativeUrl || 'App/models');
```

Reply:
Fixed